libs/sdk-ui-kit/src/Dialog/ShareDialog/ShareDialogBase/ShareGranteeBase.tsx

```diff
@@ -11,9 +11,19 @@ import { sortGranteesByName } from "./utils";
  * @internal
  */
 export const ShareGranteeBase: React.FC<IShareGranteeBaseProps> = (props) => {
-    const { grantees, owner, isDirty, onCancel, onSubmit, onGranteeDelete, onAddGranteeButtonClick } = props;
+    const {
+        isLoading,
+        grantees,
+        owner,
+        isDirty,
+        onCancel,
+        onSubmit,
+        onGranteeDelete,
+        onAddGranteeButtonClick,
+    } = props;
     const intl = useIntl();
 
+    // TODO mozna ownera pridat drive hned po nacten
```

Comment:
Please remove TODO or change it to English and add JIRA ID when it will be solved.

Reply:
Done